extensions/Oisthebestletter/sb3ai.js

```
@@ -0,0 +1,311 @@
+  worker = new Worker(workerURL, { type: "module" });
+  worker.onerror = ({ message }) => {
+    console.error("SB3.AI worker failed:", message);
+  };
+
+  URL.revokeObjectURL(workerURL);
+  workerURL = null;
+
+  return worker;
+}
+
+let requestId = 0;
```

ESLint

[error] All extension code must be within (function (Scratch) { ... })(Scratch);

(extension/iife)